PatientRecords/PatientRecords/Interfaces/IController.cs

```diff
@@ -0,0 +1,17 @@
+﻿namespace PatientRecords.Interfaces
+{
+    public interface IController
+    {
+        void Start();
```

Comment:
All of them can be async because net core 3 has an async version of main method.